index.html

```
@@ -207,14 +208,88 @@ <h2 class="sub-title" id="sub-title">지금 인기 있는 영화</h2>
+          <img src="./images/woowacourse_logo.png" width="180" />
         </p>
       </footer>
     </div>
+    <div class="modal-background active" id="modalBackground">
+      <div class="modal" id="modal">
+        <button class="close-modal" id="closeModal">
+          <img src="./images/modal_button_close.png" />
+        </button>
+        <div class="modal-container">
+          <div class="modal-image modal-skeleton-image" id="modal-image">
+            <img src="" alt="" />
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)